docs/source/index.rst

```diff
@@ -74,5 +75,6 @@ Miscellaneous
     ref/index
     howto/index
     faq
+    known-issues
```

Comment:
You didn't follow the instructions above: `.. Keep this toctree in sync with contents.rst.`